variants/NUCLEO_L476RG/PeripheralPins.c

```diff
@@ -283,3 +283,14 @@ const PinMap PinMap_CAN_TD[] = {
     {PB_9,   CAN1, STM_PIN_DATA(STM_MODE_AF_PP, GPIO_NOPULL, GPIO_AF9_CAN1)},
     {NC,    NP,    0}
 };
+
+//*** WAKE UP ***
+
+const PinMap PinMap_WKUP[] = {
```

Comment:
PeripheralPins should only contains array For Pin <--> Peripheral.
How this array is defined? Is it linked to boards? MCU ? Is it exhaustive?

Reply:
Agree with @fpistm. Please add comment ...
From what I've read in MCU datasheet that is related to a given range of MCUs (e.g. L4x6), so this can be defined for all boards using the same MCU. Whether each pin is connected or not should not change the table.